src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,113 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int START_ARRAY_CAPACITY = 16;
+    private static final double LOAD_FACTOR = 0.75;
+    private int size;
+    private double threshold;
+    private Node<K, V> currentNode;
```

Comment:
the current threshold can be calculated based on size and array length, not need to have separate variables for them, same for currentNode
```suggestion
```